src/main/java/Milo/TaskObj/TaskType.java

```diff
@@ -0,0 +1,15 @@
+package Milo.TaskObj;
+
+/*
+ * Represents the different possible task type of the task object
+ */
+public class TaskType {
+    public static enum taskType {
+        TODO,
+        EVENT,
+        DEADLINE,
+        INVALID,
+
+        DATE
```

Comment:
Reading through your code it seems like these are used to tag error to be handled by `printError` in Ui. Perhaps making it `ErrorType` instead of `TaskType`?